src/main/webapp/index.html

```diff
@@ -2,6 +2,7 @@
 <meta charset="utf-8" />
 
 <title>ZoomTube</title>
+<link rel="icon" type="image/png" href="/favicon.png" />
```

Comment:
what does fav mean?

Reply:
It's short for "favorite icon":  https://en.wikipedia.org/wiki/Favicon